src/javascript/binary/websocket_pages/user/telegram_bot.js

```diff
@@ -0,0 +1,26 @@
+const TelegramBot = (() => {
+    'use strict';
+
+    const onLoad = () => {
+        const bot_name = 'binary_test_bot';
+        const submit_button = $('#go_to_telegram');
```

Comment:
can we make a global variable within this file `const submit_button = '#go_to_telegram';` and use it on line 7 and 17 as `$(submit_button)` instead of declaring the same value twice?